support-frontend/assets/helpers/tracking/quantumMetric.ts

```diff
@@ -125,12 +126,11 @@ function sendEvent(
 		? 64
 		: 0;
 	if (window.QuantumMetricAPI?.isOn()) {
-		window.QuantumMetricAPI.sendEvent(
-			id,
-			qmCartValueEventId,
-			value,
-			payload ?? {},
-		);
+		if (payload) {
+			window.QuantumMetricAPI.sendEvent(id, qmCartValueEventId, value, payload);
+		} else {
+			window.QuantumMetricAPI.sendEvent(id, qmCartValueEventId, value);
+		}
```

Comment:
Can we replace the `if else` with:

`window.QuantumMetricAPI.sendEvent(id, qmCartValueEventId, value, payload ?? {});`

I made the change in my PR but I think it's been replaced in the rebase.

Reply:
sorry corrected now